packages/flutter/test/material/bottom_app_bar_test.dart

```diff
@@ -139,6 +139,26 @@ void main() {
     expect(physicalShape.color, const Color(0xff0000ff));
   });
 
+  testWidgets('dark theme applies an elevation overlay color', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        theme: ThemeData.from(colorScheme: const ColorScheme.dark()),
+        home: const Scaffold(
+          floatingActionButton: FloatingActionButton(
```

Comment:
Why do we want the FAB to be disabled? Doesn't matter? The FAB's color is surface when it's disabled? It would help to tie this test back to the surface color.